{
  "meta": {
    "desc": "Authentication Library for OpenStack Identity",
    "desc_long": null,
    "home": "https://github.com/openstack/keystoneauth",
    "license": {
      "deprecated": false,
      "free": true,
      "fullName": "Apache License 2.0",
      "redistributable": true,
      "shortName": "asl20",
      "spdxId": "Apache-2.0",
      "url": "https://spdx.org/licenses/Apache-2.0.html"
    },
    "maintainers": [
      {
        "email": "nix@angus.ws",
        "github": "angustrau",
        "githubId": 13267947,
        "matrix": "@angustrau:matrix.org",
        "name": "Angus Trau"
      },
      {
        "email": "5861043+superherointj@users.noreply.github.com",
        "github": "superherointj",
        "githubId": 5861043,
        "matrix": "@superherointj:matrix.org",
        "name": "S\u00e9rgio G."
      },
      {
        "email": "sandro.jaeckel@gmail.com",
        "github": "SuperSandro2000",
        "githubId": 7258858,
        "matrix": "@sandro:supersandro.de",
        "name": "Sandro J\u00e4ckel"
      }
    ]
  },
  "outputs": {
    "out": [
      "/lib/python3.8/site-packages/keystoneauth1/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/__pycache__/_fair_semaphore.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/__pycache__/_utils.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/__pycache__/adapter.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/__pycache__/discover.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/__pycache__/http_basic.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/__pycache__/noauth.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/__pycache__/plugin.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/__pycache__/service_token.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/__pycache__/session.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/__pycache__/token_endpoint.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/_fair_semaphore.py",
      "/lib/python3.8/site-packages/keystoneauth1/_utils.py",
      "/lib/python3.8/site-packages/keystoneauth1/access/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/access/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/access/__pycache__/access.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/access/__pycache__/service_catalog.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/access/__pycache__/service_providers.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/access/access.py",
      "/lib/python3.8/site-packages/keystoneauth1/access/service_catalog.py",
      "/lib/python3.8/site-packages/keystoneauth1/access/service_providers.py",
      "/lib/python3.8/site-packages/keystoneauth1/adapter.py",
      "/lib/python3.8/site-packages/keystoneauth1/discover.py",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/__pycache__/auth.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/__pycache__/auth_plugins.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/__pycache__/base.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/__pycache__/catalog.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/__pycache__/connection.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/__pycache__/discovery.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/__pycache__/http.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/__pycache__/oidc.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/__pycache__/response.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/__pycache__/service_providers.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/auth.py",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/auth_plugins.py",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/base.py",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/catalog.py",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/connection.py",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/discovery.py",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/http.py",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/oidc.py",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/response.py",
      "/lib/python3.8/site-packages/keystoneauth1/exceptions/service_providers.py",
      "/lib/python3.8/site-packages/keystoneauth1/extras/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/extras/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/extras/_saml2/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/extras/_saml2/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/extras/_saml2/__pycache__/_loading.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/extras/_saml2/_loading.py",
      "/lib/python3.8/site-packages/keystoneauth1/extras/_saml2/v3/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/extras/_saml2/v3/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/extras/_saml2/v3/__pycache__/adfs.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/extras/_saml2/v3/__pycache__/base.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/extras/_saml2/v3/__pycache__/saml2.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/extras/_saml2/v3/adfs.py",
      "/lib/python3.8/site-packages/keystoneauth1/extras/_saml2/v3/base.py",
      "/lib/python3.8/site-packages/keystoneauth1/extras/_saml2/v3/saml2.py",
      "/lib/python3.8/site-packages/keystoneauth1/extras/kerberos/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/extras/kerberos/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/extras/kerberos/__pycache__/_loading.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/extras/kerberos/_loading.py",
      "/lib/python3.8/site-packages/keystoneauth1/extras/oauth1/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/extras/oauth1/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/extras/oauth1/__pycache__/_loading.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/extras/oauth1/__pycache__/v3.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/extras/oauth1/_loading.py",
      "/lib/python3.8/site-packages/keystoneauth1/extras/oauth1/v3.py",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/__pycache__/discovery.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/__pycache__/exception.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/__pycache__/hooks.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/__pycache__/keystoneauth_betamax.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/__pycache__/plugin.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/__pycache__/serializer.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/__pycache__/v2.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/__pycache__/v3.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/discovery.py",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/exception.py",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/hooks.py",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/keystoneauth_betamax.py",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/plugin.py",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/serializer.py",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/v2.py",
      "/lib/python3.8/site-packages/keystoneauth1/fixture/v3.py",
      "/lib/python3.8/site-packages/keystoneauth1/hacking/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/hacking/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/hacking/__pycache__/checks.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/hacking/checks.py",
      "/lib/python3.8/site-packages/keystoneauth1/http_basic.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/__pycache__/access.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/__pycache__/base.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/__pycache__/v2.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/access.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/base.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/generic/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/generic/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/generic/__pycache__/base.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/generic/__pycache__/password.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/generic/__pycache__/token.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/generic/base.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/generic/password.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/generic/token.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v2.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/__pycache__/application_credential.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/__pycache__/base.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/__pycache__/federation.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/__pycache__/k2k.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/__pycache__/multi_factor.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/__pycache__/oidc.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/__pycache__/password.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/__pycache__/receipt.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/__pycache__/token.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/__pycache__/tokenless_auth.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/__pycache__/totp.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/application_credential.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/base.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/federation.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/k2k.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/multi_factor.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/oidc.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/password.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/receipt.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/token.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/tokenless_auth.py",
      "/lib/python3.8/site-packages/keystoneauth1/identity/v3/totp.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/__pycache__/_utils.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/__pycache__/adapter.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/__pycache__/base.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/__pycache__/cli.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/__pycache__/conf.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/__pycache__/identity.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/__pycache__/opts.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/__pycache__/session.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_plugins/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_plugins/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_plugins/__pycache__/admin_token.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_plugins/__pycache__/http_basic.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_plugins/__pycache__/noauth.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_plugins/admin_token.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_plugins/http_basic.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_plugins/identity/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_plugins/identity/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_plugins/identity/__pycache__/generic.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_plugins/identity/__pycache__/v2.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_plugins/identity/__pycache__/v3.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_plugins/identity/generic.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_plugins/identity/v2.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_plugins/identity/v3.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_plugins/noauth.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/_utils.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/adapter.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/base.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/cli.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/conf.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/identity.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/opts.py",
      "/lib/python3.8/site-packages/keystoneauth1/loading/session.py",
      "/lib/python3.8/site-packages/keystoneauth1/noauth.py",
      "/lib/python3.8/site-packages/keystoneauth1/plugin.py",
      "/lib/python3.8/site-packages/keystoneauth1/service_token.py",
      "/lib/python3.8/site-packages/keystoneauth1/session.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/client_fixtures.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/k2k_fixtures.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/keystoneauth_fixtures.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/matchers.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/oidc_fixtures.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/test_betamax_fixture.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/test_betamax_hooks.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/test_betamax_serializer.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/test_discovery.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/test_fair_sempahore.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/test_fixtures.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/test_hacking_checks.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/test_http_basic.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/test_matchers.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/test_noauth.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/test_service_token.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/test_session.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/test_token_endpoint.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/test_utils.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/__pycache__/utils.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/access/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/access/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/access/__pycache__/test_v2_access.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/access/__pycache__/test_v2_service_catalog.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/access/__pycache__/test_v3_access.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/access/__pycache__/test_v3_service_catalog.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/access/test_v2_access.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/access/test_v2_service_catalog.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/access/test_v3_access.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/access/test_v3_service_catalog.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/client_fixtures.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/data/README",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/data/keystone_v2_sample_request.json",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/data/keystone_v2_sample_response.json",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/data/keystone_v3_sample_request.json",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/data/keystone_v3_sample_response.json",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/data/ksa_betamax_test_cassette.yaml",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/data/ksa_serializer_data.json",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/data/test_pre_record_hook.json",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/exceptions/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/exceptions/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/exceptions/__pycache__/test_exceptions.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/exceptions/test_exceptions.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/kerberos/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/kerberos/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/kerberos/__pycache__/base.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/kerberos/__pycache__/test_fedkerb_loading.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/kerberos/__pycache__/test_kerberos_loading.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/kerberos/__pycache__/test_mapped.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/kerberos/__pycache__/test_v3.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/kerberos/__pycache__/utils.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/kerberos/base.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/kerberos/test_fedkerb_loading.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/kerberos/test_kerberos_loading.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/kerberos/test_mapped.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/kerberos/test_v3.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/kerberos/utils.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/oauth1/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/oauth1/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/oauth1/__pycache__/test_oauth1.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/oauth1/__pycache__/test_oauth1_loading.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/oauth1/test_oauth1.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/oauth1/test_oauth1_loading.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/saml2/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/saml2/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/saml2/__pycache__/test_auth_adfs.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/saml2/__pycache__/test_auth_saml2.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/saml2/__pycache__/utils.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/saml2/examples/xml/ADFS_RequestSecurityTokenResponse.xml",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/saml2/examples/xml/ADFS_fault.xml",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/saml2/fixtures/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/saml2/fixtures/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/saml2/fixtures/templates/authn_request.xml",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/saml2/fixtures/templates/saml_assertion.xml",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/saml2/fixtures/templates/soap_response.xml",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/saml2/test_auth_adfs.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/saml2/test_auth_saml2.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/extras/saml2/utils.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/__pycache__/test_access.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/__pycache__/test_identity_common.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/__pycache__/test_identity_v2.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/__pycache__/test_identity_v3.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/__pycache__/test_identity_v3_federation.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/__pycache__/test_identity_v3_oidc.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/__pycache__/test_password.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/__pycache__/test_token.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/__pycache__/test_tokenless_auth.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/__pycache__/utils.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/test_access.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/test_identity_common.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/test_identity_v2.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/test_identity_v3.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/test_identity_v3_federation.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/test_identity_v3_oidc.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/test_password.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/test_token.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/test_tokenless_auth.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/identity/utils.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/k2k_fixtures.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/keystoneauth_fixtures.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/__init__.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/__pycache__/__init__.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/__pycache__/test_adapter.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/__pycache__/test_cli.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/__pycache__/test_conf.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/__pycache__/test_entry_points.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/__pycache__/test_fixtures.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/__pycache__/test_generic.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/__pycache__/test_loading.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/__pycache__/test_session.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/__pycache__/test_v3.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/__pycache__/utils.cpython-38.pyc",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/test_adapter.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/test_cli.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/test_conf.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/test_entry_points.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/test_fixtures.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/test_generic.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/test_loading.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/test_session.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/test_v3.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/loading/utils.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/matchers.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/oidc_fixtures.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/test_betamax_fixture.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/test_betamax_hooks.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/test_betamax_serializer.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/test_discovery.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/test_fair_sempahore.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/test_fixtures.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/test_hacking_checks.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/test_http_basic.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/test_matchers.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/test_noauth.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/test_service_token.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/test_session.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/test_token_endpoint.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/test_utils.py",
      "/lib/python3.8/site-packages/keystoneauth1/tests/unit/utils.py",
      "/lib/python3.8/site-packages/keystoneauth1/token_endpoint.py",
      "/lib/python3.8/site-packages/keystoneauth1-4.4.0.dist-info/AUTHORS",
      "/lib/python3.8/site-packages/keystoneauth1-4.4.0.dist-info/INSTALLER",
      "/lib/python3.8/site-packages/keystoneauth1-4.4.0.dist-info/LICENSE",
      "/lib/python3.8/site-packages/keystoneauth1-4.4.0.dist-info/METADATA",
      "/lib/python3.8/site-packages/keystoneauth1-4.4.0.dist-info/RECORD",
      "/lib/python3.8/site-packages/keystoneauth1-4.4.0.dist-info/REQUESTED",
      "/lib/python3.8/site-packages/keystoneauth1-4.4.0.dist-info/WHEEL",
      "/lib/python3.8/site-packages/keystoneauth1-4.4.0.dist-info/direct_url.json",
      "/lib/python3.8/site-packages/keystoneauth1-4.4.0.dist-info/entry_points.txt",
      "/lib/python3.8/site-packages/keystoneauth1-4.4.0.dist-info/pbr.json",
      "/lib/python3.8/site-packages/keystoneauth1-4.4.0.dist-info/top_level.txt",
      "/nix-support/propagated-build-inputs"
    ]
  },
  "versions": [
    {
      "first": "0593b0a4de51bfdbc3e2e8be368ee9012381afad",
      "last": "020738a1fb53c0123a9a4ebd578106aa04d3ac21",
      "name": "4.4.0"
    },
    {
      "first": "02abcd6b3b31b45824cc123adecd3e03c0923740",
      "last": "03480209d911b38ba1635bbfacbdc20e89de0985",
      "name": "4.3.1"
    },
    {
      "first": "004822554c840c322153770381d90969be9a2d93",
      "last": "02b6fe2cb20a1a520c49504c4c6afde37863829e",
      "name": "4.4.0"
    },
    {
      "first": "046f81842fbe326713ba53131260d2315dcfdf75",
      "last": "0170ff4c9598c0a01d687d5241198b0a78debfd7",
      "name": "4.3.1"
    },
    {
      "first": "0379ff190e3ac1419f47153a2d1127ca620a2912",
      "last": "0379ff190e3ac1419f47153a2d1127ca620a2912",
      "name": "4.4.0"
    },
    {
      "first": "01ab36d8752c2efe901136f19c18ea309b995ac0",
      "last": "0366acbcd54e2e27d3a96c4cadd8aaf2f2ce12bf",
      "name": "4.3.1"
    }
  ]
}